src/components/Stream.tsx

```diff
@@ -243,9 +229,15 @@ export const Stream = () => {
 
     // The browser complains if we try to load a new stream without pausing first.
     // Do not immediately play the stream!
+    // we instead use a setTimeout to play the stream in the next event loop
```

Comment:
Just wondering... Video play is async, so why does it matter to wait till the next event loop to start it?

Reply:
I originally encountered errors trying immediately. The story may be different today. I can't explain it; it was too long ago